Documentation/Migrations/Hyperspace 2.0 Migration Guide.md

```diff
@@ -0,0 +1,19 @@
+# Hyperspace 2 Migration Guide
+
+This guide has been provided in order to ease the transition of existing applications using Hyperspace 1.x to the latest APIs, as well as to explain the structure of the new and changed functionality.
+
+## Requirements
+
+- iOS 8.0, tvOS 9.0, watchOS 2.0
+- Xcode 9.4
+- Swift 4.0
+
+## Overview
+
+Hyperspace 2.0 brings a ton of refinements and improvements to the simple core functionality of Hyperspace. There are a number of new APIs in Hyperspace 2, but fortunately, most is source-compatible with Hyperspace 1.x
```

Comment:
For better organization, let's separate these ideas/main changes into their own sections.